extension/src/ohlc.rs

```diff
@@ -438,7 +739,25 @@ mod tests {
                             open:(ts:\"{}\",val:1),\
                             high:(ts:\"{}\",val:1),\
                             low:(ts:\"{}\",val:1),\
-                            close:(ts:\"{}\",val:1)\
+                            close:(ts:\"{}\",val:1),\
+                            volume:Missing()\
+                            )",
+                    extreme_time, extreme_time, extreme_time, extreme_time
+                );
+                assert_eq!(expected, output.unwrap());
+
+                let stmt = format!("SELECT toolkit_experimental.candlestick(ts, price, volume)::text FROM (VALUES ('{}'::timestamptz, 1.0, 1.0)) v(ts, price, volume)", extreme_time);
```

Comment:
Can you wrap these lines?  I'm sure we could quibble over a good line length, but one of these is 202, and that's a bit much :)

Reply:
👍